src/main/java/nextstep/member/ui/AuthenticationPrincipalArgumentResolver.java

```diff
@@ -24,13 +24,24 @@ public boolean supportsParameter(MethodParameter parameter) {
     @Override
     public Object resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest, WebDataBinderFactory binderFactory) throws Exception {
         String authorization = webRequest.getHeader("Authorization");
-        if (!"bearer".equalsIgnoreCase(authorization.split(" ")[0])) {
-            throw new AuthenticationException();
-        }
+        checkTokenExists(authorization);
         String token = authorization.split(" ")[1];
+        checkValidToken(token);
 
         String email = jwtTokenProvider.getPrincipal(token);
-
         return new LoginMember(email);
     }
+
+    private void checkTokenExists(String authorization) {
```

Comment:
메서드명을 조금 더 단순하게 지어도 좋을 것 같아요
```suggestion
    private void existToken(String authorization) {
```